### ISSUE TITLE
add support for Jewel mods?

### ISSUE THREAD

**Issue by neosin** [xyzz/acquisition#162]
can you add support for jewel mods?
I find that I cannot search my jewels for things such as minion damage, weapon range, etc

**Comment by xyzz** [comment]
Is there a list of jewel mods somewhere?

**Comment by GrappigPanda** [comment]
http://pathofexile.gamepedia.com/Jewel

Is that what you're looking for by any chance, xyzz? It lists all prefixes/suffiexes.

**Comment by xyzz** [comment]
Yes, however that's way too many mods, we would need a better mod interface (for example with the search bar like how it's done at poe.trade).
